src/test/java/com/google/api/codegen/grpcmetadatagen/PackageMetadataGeneratorTest.java

```diff
@@ -81,15 +81,15 @@ protected Object run() throws Exception {
     options.set(GrpcMetadataGenerator.LANGUAGE, language);
     Map<String, Doc> generatedDocs = new GrpcMetadataGenerator(options).generate(model);
 
-    if (language == "java") {
-      return new ImmutableMap.Builder<String, Doc>().putAll(generatedDocs).build();
-    } else {
+    if (language == "python") {
```

Comment:
Well conceptually they are different... the language names in MainGapicProviderFactory are generation IDs (e.g. java_doc is not a language)

Switched to use TargetLanguage.